### DISCUSSION TITLE
Strapi register problem

### DISCUSSION THREAD

**Discussion by dominikjasek** [strapi/strapi#8474] [Legacy Topics]
Hello I am trying to use strapi API to register new user. It registers successfully (when I check it inside strapi administration panel), but as a response to request I receive this error:

```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": {},
    "data": {}
}
``` 

My request goes to localhost:1337/auth/local/register and body is this (some attributes were added by myself

```
{
    "username": "asdasd", 
    "jmeno": "adasdasd", 
    "prezdivka": "ddd", 
    "password": "aaaaaa",
    "email": "asdasd@asdasd.cz"
}
```

Can you please help me what is wrong?

**Comment by dominikjasek** [comment]
I fixed this by dsiabling email confirmation to new user.

**Reply by derrickmehaffy** [reply]
Hello,

I see you asked a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
